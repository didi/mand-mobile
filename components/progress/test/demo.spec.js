import Demo0 from './cases/demo0'
import Demo1 from './cases/demo1'
import {renderToString} from '@vue/server-test-utils'

describe('Amount', () => {
  test(`Basic`, () => {
    expect(renderToString(Demo0)).toMatchSnapshot()
  })
  test(`Other configurations`, () => {
    expect(renderToString(Demo1)).toMatchSnapshot()
  })
})
