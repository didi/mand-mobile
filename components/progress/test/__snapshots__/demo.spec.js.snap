// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress Basic 1`] = `
<div class="md-example-child md-example-child-progress">
  <div class="md-activity-indicator-rolling md-progress">
    <div class="rolling-container"><svg viewBox="0 0 110 110" preserveAspectRatio="xMidYMid" class="md-activity-indicator-svg rolling" style="width:100px;height:100px;transform:rotateZ(0deg);">
        <circle fill="none" stroke="rgba(0, 0, 0, .1)" stroke-width="5" cx="55" cy="55" r="50"></circle>
        <g class="circle">
          <circle cx="55" cy="55" fill="transparent" stroke="#2F86F6" stroke-width="5" stroke-dasharray="62.83000000000001 251.32000000000005" stroke-linecap="round" r="50" class="stroke">
            <!---->
            <!---->
          </circle>
        </g>
      </svg>
      <div class="content"><span class="progress-value">20%</span> </div>
    </div>
  </div>
  <div class="md-activity-indicator-rolling md-progress">
    <div class="rolling-container"><svg viewBox="0 0 110 110" preserveAspectRatio="xMidYMid" class="md-activity-indicator-svg rolling" style="width:100px;height:100px;transform:rotateZ(0deg);">
        <circle fill="none" stroke="rgba(0, 0, 0, .1)" stroke-width="5" cx="55" cy="55" r="50"></circle>
        <g class="circle">
          <circle cx="55" cy="55" fill="transparent" stroke="#2F86F6" stroke-width="5" stroke-dasharray="157.07500000000002 157.07500000000002" stroke-linecap="round" r="50" class="stroke">
            <!---->
            <!---->
          </circle>
        </g>
      </svg>
      <div class="content"><span class="progress-value">50%</span> </div>
    </div>
  </div>
  <div class="md-activity-indicator-rolling md-progress">
    <div class="rolling-container"><svg viewBox="0 0 110 110" preserveAspectRatio="xMidYMid" class="md-activity-indicator-svg rolling" style="width:100px;height:100px;transform:rotateZ(0deg);">
        <circle fill="none" stroke="rgba(0, 0, 0, .1)" stroke-width="5" cx="55" cy="55" r="50"></circle>
        <g class="circle">
          <circle cx="55" cy="55" fill="transparent" stroke="#2F86F6" stroke-width="5" stroke-dasharray="314.15000000000003 0" stroke-linecap="round" r="50" class="stroke">
            <!---->
            <!---->
          </circle>
        </g>
      </svg>
      <div class="content"><span class="progress-value">100%</span> </div>
    </div>
  </div>
</div>
`;

exports[`Progress Other configurations 1`] = `
<div class="md-example-child md-example-child-progress">
  <div class="md-activity-indicator-rolling md-progress">
    <div class="rolling-container"><svg viewBox="0 0 120 120" preserveAspectRatio="xMidYMid" class="md-activity-indicator-svg rolling" style="width: 100px; height: 100px; transform: rotateZ(0deg);">
        <circle fill="none" stroke="#FFF" stroke-width="10" cx="60" cy="60" r="50"></circle>
        <g class="circle">
          <circle cx="60" cy="60" fill="transparent" stroke="url(#linear)" stroke-width="10" stroke-dasharray="251.32000000000005 62.82999999999999" stroke-linecap="butt" r="50" class="stroke">
            <!---->
            <!---->
          </circle>
        </g>
        <defs>
          <linearGradient id="linear" x1="0%" y1="0%" x2="100%" y2="0%">
            <stop offset="0%" stop-color="#FF5257"></stop>
            <stop offset="100%" stop-color="#FFC541D6"></stop>
          </linearGradient>
        </defs>
      </svg>
      <div class="content"><span class="progress-value">80%</span> </div>
    </div>
  </div>
  <div class="md-activity-indicator-rolling md-progress">
    <div class="rolling-container"><svg viewBox="0 0 120 120" preserveAspectRatio="xMidYMid" class="md-activity-indicator-svg rolling" style="width: 100px; height: 100px; transform: rotateZ(-90deg);">
        <circle fill="none" stroke="rgba(0, 0, 0, .1)" stroke-width="10" cx="60" cy="60" r="50"></circle>
        <g class="circle">
          <!---->
        </g>
      </svg>
      <div class="content"><span class="progress-value"><span class="md-amount numerical">0</span>%
        </span> </div>
    </div>
  </div>
</div>
`;
